cmake/modules/gluecodium/gluecodium/Generate.cmake

```diff
@@ -88,6 +88,7 @@ function(apigen_generate)
       CPP_EXPORT
       CPP_NAMERULES
       SWIFT_NAMERULES
+      SWIFT_INTERNAL_PREFIX
```

Comment:
How about to detect prefix from target's name if not set?

Reply:
I don't think it's a good idea, there should still be an option to go with no prefix. The prefix is only needed for modularized build, maybe not even always for those. Definitely not needed for monolith builds, but makes call stacks harder to read.